symbols/arm9.yml

```
@@ -10393,11 +10401,29 @@ arm9:
         NA: 0x205F9B8
         JP: 0x205FCA8
       description: |-
-        Given a mission struct, clears some of it fields.
+        Given a mission struct, clears some of its fields.
         
         In particular, mission::status is set to mission_status::MISSION_STATUS_INVALID, mission::dungeon_id is set to -1, mission::floor is set to 0 and mission::reward_type is set to mission_reward_type::MISSION_REWARD_MONEY.
         
         r0: Pointer to the mission to clear
+    - name: ValidateNormalChallengeMission
+      address:
+        EU: 0x2060B80
+      description: |-
+        Validates a normal challenge letter mission by checking whether its client (leader), target (second member), and outlaw_backup_species (third member) match those of some mission_rescue_bin struct (alongside some other conditions).
```

Can we clarify what "normal" means?

```suggestion
        Validates a normal (non-legendary) challenge letter mission by checking whether its client (leader), target (second member), and outlaw_backup_species (third member) match those of some mission_rescue_bin struct (alongside some other conditions).
```